l2/eigen-tutorials/packages/demo-eigencall/.env-sample

```diff
@@ -5,4 +5,13 @@ DEVNET_PRIVKEY=
 L2RPC=http://127.0.0.1:8547
 
 # Set pkcs address here
-PKCS=http://localhost:3000
+PKCS=http://127.0.0.1:3000
+
+# Set TEESDK related parameters here
```

Comment:
Those envs had been moved to `tools/run.sh`.

Reply:
OK, I'll remove them in the next commit.